design/design.md

```diff
@@ -0,0 +1,112 @@
+Applications?
+mithril, peras, leios
+
+What cryptographic primitives will we implement?
+I propose that we implement the generic ALBA primitive to make it as applicable
+as possible.
+This means there is only a generic notion of "elements" and no notions of
```

Comment:
We should clarify what it means to be an "element"? Just a bunch of bytes as suggested by @rrtoledo ?

Reply:
I think it is best to keep it as bytes so that Alba is protocol agnostic.